__tests__/unit/http_adapter.test.ts

```diff
@@ -239,6 +240,42 @@ describe('HttpAdapter', () => {
     }
   })
 
+  describe('User-Agent', () => {
+    class MyTestHttpAdapter extends BaseHttpAdapter {
+      constructor(application_id?: string) {
+        super(
+          { application_id } as ConnectionParams,
+          new TestLogger(),
+          {} as Http.Agent
+        )
+      }
+      protected createClientRequest(): Http.ClientRequest {
+        return {} as any
+      }
+      public getDefaultHeaders() {
+        return this.headers
+      }
+    }
+
+    it('should have proper user agent without app id', async () => {
+      const myHttpAdapter = new MyTestHttpAdapter()
+      const headers = myHttpAdapter.getDefaultHeaders()
+      expect(headers['Authorization']).toMatch(/^Basic [A-Za-z0-9/+=]+$/)
```

Comment:
What is the purpose of this assertion in `User-Agent` test?

Reply:
let's move it to a separate test case then